website/project/metadata/schemas.py

```diff
@@ -38,5 +39,5 @@ def from_json(fname):
     'OSF-Standard Pre-Data Collection Registration',
```

Comment:
People will start building registries on the OSF a la pre-reg but via their own interfaces. We can re-architect to make that really easy--perhaps with the inclusion of JamDB buckets on GUIDs.